build.gradle

```diff
@@ -10,7 +10,7 @@ ext.dropwizard_version="4.1.2"
 ext.OWASP_SANITIZER_VERSION="20180219.1"
 ext.JAXB_API_VERSION="2.2.12"
 ext.JAXB_IMPL_VERSION="2.2.11"
-ext.JAVALIN_VERSION='4.0.1'
+ext.JAVALIN_VERSION='4.1.1'
```

Comment:
I very much appreciate that we are actively keeping this up-to-date instead of lingering.